commands/keplr.js

```diff
@@ -273,6 +272,7 @@ const keplr = {
   },
 
   async switchWallet({ walletName }) {
+    const originalURL = (await playwright.keplrWindow()).url();
```

Comment:
I am all in for sending params through an object. But if this method is going to have a single param, in that case, I feel it's best to pass it without an object to ensure clarity and consistency with other methods. 

Also, we should use objects for sending params in `importWallet` flow. There we have multiple params and it sometimes becomes tedious to maintain order while passing params.